### DISCUSSION TITLE
docker 安装, 使用 host 模式不能访问设置页面

### DISCUSSION THREAD

**Discussion by scott-topview** [jeessy2/ddns-go#884] [Q&A]
Debian 12 , 使用 docker 安装, 使用 host 模式启动, 显示已经启动,但是不能访问设置页面.
使用 docker ps 查看, 没有 PORTS.

使用非 host 模式启动, 也可启动成功, 但是无法用命令获取 IPv6
求解答

**Comment by jeessy2** [comment]
检查是不是防火墙拦截访问了
host那个自行搜索

**Reply by scott-topview** [reply]
查了防火墙, 确实是! 非常感谢
`ufw allow 9876` 添加端口, 可以访问设置页面了.
